db/seeds.rb

```diff
@@ -39,7 +39,9 @@
     FactoryBot.create(:investment, :random, user:, date: Date.current - index)
   end
 
-  # p 'Creating Exchanges...'
+  # p 'Creating TransactionEntities...' do
```

Comment:
I decided not to create TransactionEntity instances on `seeds.rb` because we don't have Exchange yet.